### PR TITLE
Add network logs to TOML tests

### PR DESCRIPTION
**Pull request by vitchor**

### CODE REVIEW

File: cases/toml.test.js [hunk 1]
```diff
@@ -30,10 +38,16 @@ describe("TOML File", () => {
...
     let fileSize;
+    let logs;
     beforeAll(async () => {
-      const response = await fetch(url + ".well-known/stellar.toml");
+      const { response, json: text, logs: outputLogs } = await loggableFetch(
+        url + ".well-known/stellar.toml",
+        {},
+        false,
+      );
+      logs = outputLogs;
```
Comment (vitchor) [inline cases/toml.test.js:48]: Any ideas on how to make this cleaner?

File: cases/util/loggableFetcher.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
-export const loggableFetch = async (requestURL, requestDictionary) => {
+export const loggableFetch = async (
+  requestURL,
+  requestDictionary = {},
+  responseIsJSON = true,
+) => {
   const response = await fetch(requestURL, requestDictionary);
-  const json = await response.json();
+
+  const json = responseIsJSON ? await response.json() : await response.text();
```
Comment (vitchor) [inline cases/util/loggableFetcher.js:10]: This is not great, as the response body may not be a JSON.

File: cases/toml.test.js [hunk 3]
```diff
@@ -8,13 +8,21 @@ const url = urlBuilder.toString();
 
 describe("TOML File", () => {
   it("exists", async () => {
-    const response = await fetch(url + ".well-known/stellar.toml");
-    expect(response.status).toBe(200);
+    const { status, logs } = await loggableFetch(
+      url + ".well-known/stellar.toml",
+      {},
```
Comment (vitchor) [inline cases/toml.test.js:13]: Not really clean to send the "{}". How can I use the default value from loggableFetch?

File: cases/util/loggableFetcher.js (hunk 2 above)
Comment (msfeldstein) [reply cases/util/loggableFetcher.js:10]: i think its fine to just not use this for the toml tests, they're pretty simple and we don't need any complex inspection for it do we?

Comment (vitchor) [reply cases/util/loggableFetcher.js:10]: Yeah, I guess you're right 😢

**Pull request closed without merging**